mne/viz/utils.py

```diff
@@ -2510,20 +2516,23 @@ def _check_cov(noise_cov, info):
 
 
 def _set_title_multiple_electrodes(title, combine, ch_names, max_chans=6,
-                                   all=False, ch_type=None, ):
+                                   all=False, ch_type=None):
     """Prepare a title string for multiple electrodes."""
     if title is None:
         title = ", ".join(ch_names[:max_chans])
-        if ch_type is not None:
-            ch_type = " " + ch_type[0].upper() + ch_type[1:]
+        ch_type = _channel_type_prettyprint.get(ch_type, ch_type)
+        if ch_type is None:
+            ch_type = "sensor"
+        if len(ch_names) > 1:
+            ch_type += "s"
```

Comment:
For future reference you can just do `ch_type += _pl(len(ch_names))` (from `mne.utils`). It also works with `len(x) == 0`, which this will not, though it won't matter here presumably